symfony/http-kernel/CVE-2015-4050.yaml

```diff
@@ -14,4 +14,7 @@ branches:
     2.6.x:
         time:     2015-05-26 23:55:51
         versions: [>=2.6.0,<2.6.8]
+    2.7.x:
+        time:     2015-05-26 23:55:51
+        versions: [>=2.7.0-dev,<2.7.0]
```

Comment:
well, it looks like semver.mwl.be does not consider than a beta version is below the corresponding stable version: http://semver.mwl.be/#?package=symfony%2Fsymfony&version=%3C2.7.0&minimum-stability=dev does not match the 2.7.0 beta releases while it should.